deploy/grafana_dashboard.json

```diff
@@ -0,0 +1,927 @@
+        "y": 26
+      },
+      "id": 22,
+      "options": {
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "showThresholdLabels": false,
+        "showThresholdMarkers": true
+      },
+      "pluginVersion": "9.1.0",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "czgrlG4Vk"
+          },
+          "editorMode": "code",
+          "expr": "(sum(rate(ovms_inference_time_us_sum{name=\"$model_name\", version=\"$model_version\"}[1m])) / 1000) / sum(rate(ovms_inference_time_us_count{name=\"$model_name\", version=\"$model_version\"}[1m]))",
+          "legendFormat": "inference_time",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "Mean Inference Time [ms]",
+      "type": "gauge"
+    },
+    {
+      "datasource": {
+        "type": "prometheus",
+        "uid": "czgrlG4Vk"
+      },
+      "description": "Mean time of a request waiting for inference execution",
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "thresholds"
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green"
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": [
+          {
+            "matcher": {
+              "id": "byName",
+              "options": "queue_time"
+            },
+            "properties": [
+              {
+                "id": "color",
+                "value": {
+                  "fixedColor": "orange",
+                  "mode": "fixed"
+                }
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 12,
+        "y": 26
+      },
+      "id": 23,
+      "options": {
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "lastNotNull"
+          ],
+          "fields": "",
+          "values": false
+        },
+        "showThresholdLabels": false,
+        "showThresholdMarkers": true
+      },
+      "pluginVersion": "9.1.0",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "czgrlG4Vk"
+          },
+          "editorMode": "code",
+          "expr": "(sum(rate(ovms_wait_for_infer_req_time_us_sum{name=\"$model_name\", version=\"$model_version\"}[1m])) / 1000) / sum(rate(ovms_wait_for_infer_req_time_us_count{name=\"$model_name\", version=\"$model_version\"}[1m]))",
+          "legendFormat": "queue_time",
+          "range": true,
+          "refId": "A"
+        }
+      ],
+      "title": "Mean Time in Queue [ms]",

```

Comment:
Maybe more descriptive? Mean wait time in Infer Requests Queue? Mean wait time for infer request?